Loop/Managers/ServicesManager.swift

```diff
@@ -23,17 +23,21 @@ class ServicesManager {
 
     private let servicesLock = UnfairLock()
 
+    weak var dataManager: LoopDataManager?
```

Comment:
Consider calling this `loopDataManager` (since there's also a `DeviceDataManager`)